### PR TITLE
Add methods on Task to get all committers and reviewers

### PR DESCRIPTION
**Pull request by wlonk**
cat poking head up through a hole in the pavement

### CODE REVIEW

**Comment by wlonk** [comment]
[As an Admin, I want to see a list of all users who contributed to a given Task.](https://trello.com/c/ZIazywAF/209-as-an-admin-i-want-to-see-a-list-of-all-users-who-contributed-to-a-given-task)

**Comment by wlonk** [comment]
@jgerigmeyer let me know what else you think this needs; I suspect we want to at least surface this info in the admin interface, but before I do that, let me know if I'm missing anything else.

**Comment by wlonk** [comment]
@jgerigmeyer ready for actual review.

File: metecho/api/hook_serializers.py [hunk 1]
```diff
@@ -32,9 +32,10 @@ class PrBranchSerializer(serializers.Serializer):
 
 
 class PrSerializer(serializers.Serializer):
-    merged = serializers.BooleanField()
+    merged = serializers.BooleanField(required=False)
```
Comment (jgerigmeyer) [inline metecho/api/hook_serializers.py:35]: @wlonk Apparently `pull_request_review` events have `merged_at`, but no `merged` key. This works, but feel free to handle it differently if you like.

File: metecho/api/admin.py [hunk 2]
```diff
@@ -115,6 +115,25 @@ class ProjectSlugAdmin(admin.ModelAdmin):
...
+        ("pr_number", "pr_is_open"),
+        ("review_submitted_at", "review_valid", "review_status", "review_sha"),
+        ("reviewers", "get_all_users_in_commits"),
+        "status",
+        ("assigned_dev", "assigned_qa"),
+    )
+    readonly_fields = (
+        "commits",
+        "reviewers",
+        "get_all_users_in_commits",
```
Comment (jgerigmeyer) [inline metecho/api/admin.py:135]: It's not very pretty, but it's there! 🎉  This is fine for now, until we know how we want to access it.

Screenshot 2020-07-16 18 37 59

**Comment by jgerigmeyer** [comment]
@wlonk I'll let you merge, once you've had a chance to review my changes.

**Pull request merged**